include/zephyr/toolchain/common.h

```diff
@@ -143,8 +143,10 @@
  */
 #ifndef ZTEST_UNITTEST
 #define __syscall static inline
+#define __syscall_always_inline static inline __attribute__((always_inline))
 #else
 #define __syscall
+#define __syscall_always_inline
 #endif /* ZTEST_UNITTEST */
```

Comment:
No, because I think this comment is also valid also for `__syscall_always_inline` https://github.com/zephyrproject-rtos/zephyr/blob/2236e31f6a49f9fb4f64cc1ebace010f1c0d86a9/include/zephyr/toolchain/common.h#L138-L142